plopfile.js

```diff
@@ -61,6 +61,19 @@ module.exports = function(plop) {
           'packages/{{kebabCase name}}/src/test/{{properCase name}}.test.ts',
         templateFile: 'templates/test.hbs.ts',
       },
+      {
+        type: 'modify',
+        path: 'packages/tsconfig.json',
+        transform: (file, {name}) => {
+          const tsConfig = JSON.parse(file);
+          tsConfig.references.push({path: `./${name}`});
```

Comment:
I think the package name needs to be kebab case. If you write the name correctly like `react-foo` this will work. But if you do `react foo` the package directory will be named `react-foo` and the tsconfig will add `react foo` as reference.

Maybe we don't want to support that 🤷🏼‍♂️

Reply:
Good point. Now forcing kebab case with `plop.getHelper('kebabCase')`.